packages/macros/tests/babel/macro-condition.test.ts

```diff
@@ -401,11 +402,11 @@ describe('macroCondition', function () {
           expect(code).not.toMatch(/alpha/);
         });
 
-        runTimeTest('can be used as class field initializer', () => {
+        runTimeTest('given runtime implementation, it can be used as class field initializer', () => {
```

Comment:
is there a runtime test exercising _just_ `isTesting`?

Reply:
Yeah, there is something already [here](https://github.com/embroider-build/embroider/blob/main/packages/macros/tests/babel/env-macros.test.ts#L76-L85)